src/mobile/apps/artwork/client/index.coffee

```diff
@@ -8,8 +8,9 @@ ArtworkPartnerView = require '../components/partner/view.coffee'
 Artwork = require '../../../models/artwork.coffee'
 Artworks = require '../../../collections/artworks.coffee'
 fetchArtworkBuckets = require '../components/related_artworks/index.coffee'
-{ recordArtworkView } = require 'lib/components/record_artwork_view'
-{ enableIntercom } = require 'lib/intercom'
+{ recordArtworkView } = require '../../../../lib/components/record_artwork_view'
+{ enableIntercom } = require '../../../../lib/intercom'
```

Comment:
This whole app can be deleted, but not suure why this wasn't erring out before; coffeescript doesn't work with absolute paths.

Reply:
If webpack was handling the module resolution it should handle aliased paths just fine. Coffeescript itself should really do anything with the `require` statements here... At least I think? 🤔

Reply:
Yah, that seems to reason but we've noticed interop problems involving absolute paths and coffee / js files a few times in the past. (Ignoring server-side code which goes through babels path rewriting mechanism, and doesn't know how to handle coffeescript.)